pom.xml

```diff
@@ -93,7 +93,7 @@
         <version.surefire.plugin>2.19.1</version.surefire.plugin>
         <version.war.plugin>2.6</version.war.plugin>
         <version.fabric8.plugin>3.1.83</version.fabric8.plugin>
-        <version.liberty.plugin>1.2.1</version.liberty.plugin>
+        <version.liberty.plugin>2.0</version.liberty.plugin>
 
```

Comment:
What's the plugin update for?

Reply:
just to be on the latest and greatest I suppose.  I believe there were some big fixes in here that will be necessary if we ever want to run this with OpenLiberty instead of WAS Liberty also.